### PR TITLE
issue-file fix: correct *BinaryData

### PR DESCRIPTION
**Pull request by Gi729sdF5hdf**

### CODE REVIEW

File: formatter/json_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		require.NoError(t, err)
+		resp := w.Header().Get("Content-Type")
+		require.Equal(t, contentType, resp)
+	}
+
+	w := httptest.NewRecorder()
+	handler(w, nil)
+}
+
+func TestJsonWrite_Error(t *testing.T) {
```
Comment (raoptimus) [inline formatter/json_test.go]: why should the code throw an error?

File: formatter/json_test.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		require.NoError(t, err)
+		resp := w.Header().Get("Content-Type")
+		require.Equal(t, contentType, resp)
+	}
+
+	w := httptest.NewRecorder()
+	handler(w, nil)
+}
+
+func TestBinaryWrite_Error(t *testing.T) {
```
Comment (raoptimus) [inline formatter/json_test.go:30]: why should the code throw an error?

Comment (Gi729sdF5hdf) [reply formatter/json_test.go:30]: because the function will not return a binary file

File: formatter/json_test.go (hunk 1 above)
Comment (Gi729sdF5hdf) [reply formatter/json_test.go]: not actual

**Pull request merged**